packages/immutadot/src/util/UsingWrapper.js

```diff
@@ -52,8 +67,8 @@ class UsingWrapper {
    */
   _call(fn, object, path, pArgs) {
     let callArgs = pArgs
-    const args = concat(
-      map(this._paths, usingPath => {
+    const args = [].concat(
+      this._paths.map(usingPath => {
```

Comment:
`this._paths` is guaranted to be an array, and concat doesn't mutate, call `this._paths.concat`